appstore/utils/validator/file_validator.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import UploadFile
+
+from appstore.exceptions.appstore_exceptions import InvalidFile
+
+
+def file_is_valid(file: UploadFile) -> bool:
+    if file is None:
+        raise InvalidFile
+
+    if file.content_type.capitalize() != "image/jpeg":
```

Comment:
Do you allow only jpeg files?

Reply:
Our database does not have a field for filename nor file type. Please escalate the issue to our DevOps team.

Reply:
that's your problem

Reply:
The same request has been already submitted. I'm closing this one.